### ISSUE TITLE
Mention of ETA while training data

### ISSUE THREAD

**Issue by rahul-raj** [deeplearning4j/deeplearning4j#5550]
I'm not sure whether DL4J currently handle this, otherwise it will be a new feature request. It would be nice if we have an ETA mentioned while we train the data, especially it will be helpful if there are lots of data with more epochs. 
Either we can show ETA on console (along with ScoreIterationListener) or display in DL4J UI.

**Comment by raver119** [comment]
We could do that for multiepoch training i think, but for single (or first) epoch that’s impossible

**Comment by rahul-raj** [comment]
@raver119 
Yeah, it would be convenient to have this for multiepoch training. 
Do you mean to say that it's already existing functionality in dl4j?

**Comment by AlexDBlack** [comment]
We already have this:
https://github.com/deeplearning4j/deeplearning4j/blob/master/deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/optimize/listeners/TimeIterationListener.java

It's also listed in the quick ref: https://deeplearning4j.org/quickref#listeners

I'll close this - though if that's not suitable, we can re-open.

**Comment by agibsonccc** [comment]
We don't have it for epochs though. I'd also suggest putting this in the UI.

**Comment by raver119** [comment]
For UI that would be definitely ok, but for regular training - console spam isn't something we want i bet. So we should stick to listeners for that. But UI - yes, definitely we can add that, i.e. based on first epoch knowledge.
